src/swarm/neo/client/ConnectionSet.d

```diff
@@ -917,12 +918,8 @@ version (UnitTest)
 unittest
 {
     static void setAddress ( ref AddrPort address, ubyte[] address_bytes ... )
-    in
```

Comment:
Are these changes related to the D2 transition?

Reply:
This was mentioned in the D2-migration docs [here](https://github.com/sociomantic/backend/blob/master/doc/d2-migration/stage3.md#invariants).